server/main.go

```diff
@@ -3,31 +3,20 @@ package main
 //go:generate bash -c "protoc --experimental_allow_proto3_optional --proto_path=../protobuf --go_out=generated/ --go-grpc_out=generated/ $(find ../protobuf -type f -name *.proto)"
 
 import (
-	"context"
 	"fmt"
 	"log"
 	"net"
 	"os"
 	"strconv"
 
-	gppb "github.com/MettyS/checkers/server/generated"
+	pb "github.com/MettyS/checkers/server/generated"
+	"github.com/MettyS/checkers/server/services"
 	"google.golang.org/grpc"
 )
 
 type server struct {
 }
 
-func (s *server) MakeMoves(ctx context.Context, req *gppb.MoveRequest) *gppb.MoveResponse {
-
-	fmt.Println("MakeMoves Running!")
-
-	res := &gppb.MoveResponse{
-		MoveSuccess: true,
-	}
-
-	return res
-}
-
 // BoardUpdateSubscription
 
 // rpc MakeMoves(MoveRequest) returns (MoveResponse) {}
```

Comment:
These comments weren't added before in this changeset, but they should be removed eventually

Reply:
Removed!